apps/meteor/app/apps/server/bridges/listeners.ts

```
@@ -0,0 +1,366 @@
+				if (!department) {
+					throw new Error(`Department ${data} not found`);
+				}
+
+				return this.orch.getManager().getListenerManager().executeListener(inte, { department });
+			}
+
+			default:
+				const room = await this.orch
+					.getConverters()
+					.get('rooms')
+					.convertRoom(data as IRoom);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)